testing/libtss2/APKBUILD

```diff
@@ -0,0 +1,90 @@
+# Maintainer: Erik Larsson <who+alpine@cnackers.org>
+# Contributor: Erik Larsson <who+alpine@cnackers.org>
+pkgname="libtss2"
+pkgver="2.3.1"
+pkgrel=0
+pkgdesc="TPM 2.0 TSS"
+license="BSD-2-Clause"
+url="https://github.com/tpm2-software/tpm2-tss/"
+arch="all"
+makedepends="openssl-dev doxygen perl linux-headers cmocka-dev"
+subpackages="$pkgname-doc $pkgname-mu $pkgname-sys $pkgname-esys
+$pkgname-tcti-device $pkgname-tcti-mssim $pkgname-rc $pkgname-tctildr
+$pkgname-static $pkgname-dev"
+source="https://github.com/tpm2-software/tpm2-tss/releases/download/$pkgver/tpm2-tss-$pkgver.tar.gz"
+builddir="$srcdir/tpm2-tss-${pkgver}"
+
+build() {
+    # Disable inlining so tctildr unit tests will work
+    # gcc inlines strlen which the cmocka tests tries to wrap
+    ./configure CFLAGS="-fno-inline" \
+		--prefix=/usr \
+		--without-udevrulesdir \
+		--with-tctidefaultmodule="libtss2-tcti-device.so" \
+		--with-tctidefaultconfig=/dev/tpmrm0 \
+		--with-crypto=ossl \
+		--enable-unit
+    make
+}
+
+check() {
+    make check
+}
+
+package() {
+    make DESTDIR="$pkgdir" install
+}
+
+mu() {
+    pkgdesc="TPM 2.0 Marshaling libraries"
+    mkdir -p "$subpkgdir/usr/lib"
+    mv "$pkgdir"/usr/lib/libtss2-mu.so.* "$subpkgdir/usr/lib"
+}
+
+sys() {
+    pkgdesc="TPM 2.0 System API libraries"
+    mkdir -p "$subpkgdir/usr/lib"
+    mv "$pkgdir"/usr/lib/libtss2-sys.so.* "$subpkgdir/usr/lib"
+}
+
+esys() {
+    pkgdesc="TPM 2.0 Enhanced System API libraries"
+    mkdir -p "$subpkgdir/usr/lib"
+    mv "$pkgdir"/usr/lib/libtss2-esys.so.* "$subpkgdir/usr/lib"
+}
+
+device() {
+    pkgdesc="TPM 2.0 Device TCTI"
+    provides="libtss2-tcti"
+    provider_priority="100"
+    mkdir -p "$subpkgdir/usr/lib"
+    mv "$pkgdir"/usr/lib/libtss2-tcti-device.so* "$subpkgdir/usr/lib"
+}
+
+mssim() {
+    pkgdesc="TPM 2.0 Simulator TCTI"
+    provides="libtss2-tcti"
+    provider_priority="10"
+    mkdir -p "$subpkgdir/usr/lib"
+    mv "$pkgdir"/usr/lib/libtss2-tcti-mssim.so* "$subpkgdir/usr/lib"
+}
+
+rc() {
+    pkgdesc="TPM 2.0 RC libraries"
+    mkdir -p "$subpkgdir/usr/lib"
+    mv "$pkgdir"/usr/lib/libtss2-rc.so* "$subpkgdir/usr/lib"
+}
+
+tctildr() {
+    pkgdesc="TPM 2.0 TCTI loader libraries"
+    mkdir -p "$subpkgdir/usr/lib"
+    mv "$pkgdir"/usr/lib/libtss2-tctildr.so* "$subpkgdir/usr/lib"
+}
+
+static() {
+    pkgdesc="TPM 2.0 (static libraries)"
+    mkdir -p "$subpkgdir/usr/lib"
+    mv "$pkgdir"/usr/lib/*.a "$subpkgdir/usr/lib"
+}
```

Comment:
abuild should do this automatically for you if you add $pkgname-static to subpackages